getting_started/view-a-report.rst

```diff
@@ -1,3 +1,7 @@
+.. meta::
+    :description: A quick overview of how to view the report of last test run. It guides you through how to view report of last execution on different os, in different languages, and on different ides...
```

Comment:
A quick overview of how to view test reports. It guides you through how to view report of last execution on different os, in different languages, and on different ides..